warmup/summary_statistics.py

```diff
@@ -410,9 +417,8 @@ def write_html_table(summary_data, html_filename):
             else:
                 mean_steady_iter = ''
             if bmark['steady_state_time'] is not None:
-                mean_steady = '%.5f (%.5f, %.5f)' % (bmark['steady_state_time'],
-                                                     bmark['steady_state_time_iqr'][0],
-                                                     bmark['steady_state_time_iqr'][1])
+                mean_steady = '%.5f&plusmn;%.6f' % (bmark['steady_state_time'],
```

Comment:
yes, but only for the **steady state perf** columns of the tables. This was the cause of the HTML table script crashing in the master branch.